airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -438,14 +439,20 @@ private JdbcDatabase createDatabase(JsonNode config) {
 
   protected static class TableInfo {
 
+    private final String schemaName;
     private final String name;
     private final List<Field> fields;
 
-    public TableInfo(String name, List<Field> fields) {
+    public TableInfo(String schemaName, String name, List<Field> fields) {
```

Comment:
does this all work okay with MySQL (which doesn't have schemas)?

Reply:
Yes, I think we already figured it out when we made the PR to discover tables from another schema than just the `public` one.

Reply:
kk